Components/LandingNav.tsx

```diff
@@ -2,19 +2,19 @@ import Link from "next/link";
 import { useRouter } from "next/router";
 import { DarkModeToggle } from "../Components/components";
 
-const LandingNav = () => {
+const LandingNav = ({ pageNotFound }: { pageNotFound: boolean }) => {
```

Comment:
```suggestion
const LandingNav = ({ pageNotFound = false }: { pageNotFound: boolean }) => {
```

By default keep it `false`